### PR TITLE
Add `atanh` in tensorflow frontend

### LINKED ISSUE #13291: atanh

### PR DESCRIPTION
**Pull request by Smit-create**
Closes #13291

### CODE REVIEW

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 1]
```diff
@@ -723,6 +723,60 @@ def test_tensorflow_asinh(
     )
 
 
+# acosh
+@handle_frontend_test(
+    fn_tree="tensorflow.math.acosh",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
[the remark this answers is not in the document]
Comment (Smit-create) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py]: Thanks @AnnaTz, I tried looking into the failures on the CI and couldn't find failures related to this PR. Can you please have a look?

[the remark this answers is not in the document]
Comment (AnnaTz) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py]: I couldn't find failures either. This is just a matter of good coding practice in our repository. Right now you still haven't add the supported dtypes decorator as described in the link I attached above.

File: ivy/functional/frontends/tensorflow/math.py [hunk 2]
```diff
@@ -49,6 +49,24 @@ def asinh(x, name="asinh"):
     return ivy.asinh(x)
 
 
+@with_supported_dtypes(
+    {"2.11.0 and below": ("bfloat16", "float32", "float64", "complex64", "complex128")},
```
Comment (AnnaTz) [inline ivy/functional/frontends/tensorflow/math.py:53]: I think float16 (half) is also supported.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py [hunk 3]
```diff
@@ -723,6 +723,60 @@ def test_tensorflow_asinh(
     )
 
 
+# acosh
+@handle_frontend_test(
+    fn_tree="tensorflow.math.acosh",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple([ivy.float32, ivy.float64]),
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py:730]: Also, given that the functions have the proper dtype decorator, the dtypes for the tests should be specified as `available_dtypes=helpers.get_dtypes("valid")`.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py (hunk 3 above)
Comment (Smit-create) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py]: Thanks, changed that.

**Comment by Smit-create** [comment]
I have removed the `acosh` because checking the Issue OP in #1545 , I think someone later picked `acosh` and wasn't aware of this open PR.

**Comment by AnnaTz** [comment]
> I have removed the `acosh` because checking the Issue OP in #1545 , I think someone later picked `acosh` and wasn't aware of this open PR.

Alright, I'll wait for the CI test to run. Other than that looks good to me.

**Comment by AnnaTz** [comment]
There seems to be something wrong, the CI does not detect your function. Please try pulling the current master to your fork's master.

**Comment by Smit-create** [comment]
@AnnaTz  Is there any specific reason related to this PR for test failures?

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**